lib/publishing/nuget/publish.sh

```diff
@@ -4,6 +4,17 @@ set -eu # we don't want "pipefail" to implement idempotency
 echo "Installing required CLI tools: jq, openssl..."
 yum install -y jq openssl
 
+if [[ "${FOR_REAL:-}" == "true" ]]; then
+    dotnet=dotnet
```

Comment:
Doesn't this require the use of `$dotnet` somewhere down the line?

Reply:
D'oh! Of course it does. Fixed.